src/main/java/com/example/gradecalculator/entities/SchoolYear.java

```diff
@@ -26,7 +26,7 @@ public class SchoolYear {
     @OneToMany(mappedBy = "schoolYear")
     private List<Subject> subjects;
 
-    public SchoolYear() {
+    public SchoolYear(long l, String s, GradeSystem gradeSystem) {
     }
 
     public SchoolYear(String name, LocalDate startDate, LocalDate endDate, GradeSystem gradeSystem) {
```

Comment:
also a constructor that has parameters but does only assign gradeSystem

Reply:
but other values are never assigned. use constructor with only gradeSystem instead